styles.css

```diff
@@ -1022,3 +1027,610 @@
+  }
+
+  h2 {
+    margin-top: 0%;
+  }
+
+  .snap1 {
+    display: none;
+  }
+
+  .left-block {
+    margin: 0% 2%;
+    padding: 0% 2%;
+  }
+
+  #desk {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+  }
+
+  .pt {
+    margin-top: 0%;
+  }
+
+  .tags-l {
+    display: flex;
+    padding: 0;
+    gap: 8px;
+    width: 99%;
+    margin-top: 8%;
+  }
+
+  .tag1 {
+    height: 16px;
+    width: 27px;
+  }
+
+  .tag12 {
+    display: flex;
+    align-items: center;
+    width: 44%;
+    height: 22px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .frame2 {
+    width: 100%;
+    gap: 15px;
+  }
+
+  #br {
+    display: none;
+  }
+
+  .tag2 {
+    width: 10%;
+    height: 15px;
+  }
+
+  .tag3 {
+    width: 31%;
+    height: 15px;
+  }
+
+  .myself {
+    width: 60%;
+    height: 80px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 40px;
+    display: flex;
+    align-items: center;
+  }
+
+  .html {
+    width: 29px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .css {
+    width: 21px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .ruby {
+    width: 100%;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .js {
+    width: 65px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .pry-text {
+    width: 88%;
+    height: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 0%;
+  }
+
+  .see-project-l {
+    width: 47%;
+    margin-top: 3%;
+  }
+
+  .snap2 {
+    display: none;
+  }
+
+  #desk2 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+    order: 1;
+  }
+
+  #desk3 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+  }
+
+  #desk4 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+    order: 1;
+  }
+
+  .snap3 {
+    display: none;
+  }
+
+  .snap4 {
+    display: none;
+  }
+
+  .about-myself {
+    flex-direction: row;
+    padding-bottom: 20%;
+  }
```

Comment:
- [ ] Good job translating the mobile version into a desktop version, however the rule stays the same :100: , your design should stick to the Figma design , that means that position m margin , padding, width should be the same as in the Figma design :+1:. Kindly make sure the `about` section sticks to the Figma design :smile: 

![Screenshot from 2023-03-02 10-54-01](https://user-images.githubusercontent.com/80458199/222382963-d100f7e1-53fa-47e3-bedb-4b9e0dfaa4cc.png)